context/app/static/js/components/Search/__tests__/SearchWrapper.spec.js

```diff
@@ -0,0 +1,34 @@
+import { getByPath, getOrder } from '../SearchWrapper';
+
+describe('SearchWrapper helper functions', () => {
+  describe('getByPath', () => {
```

Comment:
Nesting describe can become confusing and cluttered. I think separating tests into files instead of using `describe` is more clear, but this is an extension of my preference to have more files instead of longer files which you may not share.  I've also been using `test` instead of `it` when not inside of `describe`. I am fine with using both, but wanted to point that out in case you prefer to be consistent.

Reply:
no strong preference... I'll split this up.